clients/cli/src/main.rs

```diff
@@ -135,9 +136,19 @@ fn delete(core: &Core, target: &str, force: bool) -> Result<(), CliError> {
 
     if !force {
         let mut phrase = format!("delete '{}'", target);
+
         if f.is_folder() {
-            let count = core.get_and_get_children_recursively(f.id)?.len();
-            phrase = format!("{phrase} and its {count} children")
+            let count = core
+                .get_and_get_children_recursively(f.id)
+                .unwrap_or_default()
+                .len() as u64
+                - 1;
+
+            phrase = match count.cmp(&1) {
+                Less => format!("{phrase}"),
+                Greater => format!("{phrase} and its {count} children"),
+                Equal => format!("{phrase} and its 1 child"),
+            };
```

Comment:
If this necessitated a `match`, I'd just say no op the the `Less` case, but it doesn't. This is simply just and `if {} else if {}`. No `cmp`ing or `Ordering` necessary.